packages/app-desktop/gui/ConfigScreen/controls/plugins/PluginsStates.tsx

```diff
@@ -98,6 +100,11 @@ export default function(props: Props) {
 	const [fetchManifestTime, setFetchManifestTime] = useState<number>(Date.now());
 
 	const pluginService = PluginService.instance();
+	const store = app().store();
+
+	useEffect(()=>{
+		setSearchQuery(store.getState().pluginSearchQuery);
+	},[store.getState().pluginSearchQuery]);
 
```

Comment:
You shouldn't access the state that way. Instead pass the state as props to the component.